vignettes/parallel.Rmd

```diff
@@ -0,0 +1,218 @@
+
+## Memory requests
+
+Two methods are currently provided for specifying memory usage. These can be specified as an integer number of gigabytes, or alternative as strings such as `"64G`" or `"1T"` to represent 64Gb, or 1Tb respectively.
+
+* The `memory_per_node` specifies very simply that your task should only be run on a node that has at least that much memory. Remember that the node's memory will be shared between all the tasks running on that node, so you could also consider specifying `cores = Inf`, or `exclusive = TRUE` if you think you'll need the whole node's memory to yourself.  
+
+* If you are launching many tasks, and know the maximum memory your task needs, then you can specify `memory_per_process` to tell the cluster about that. The cluster will then  avoid allocating too many of your tasks to the same node, if the combined memory needed by those tasks will exceed what the node has. This can't really be guaranteed, unless everyone agrees to set `memory_per_process`, but it should help in the common case where your own tasks might be stacked up on a node together.
+
+## Running on Particular nodes
+
+At present, the nodes on the new cluster are all very similar to each other. This may change over time, and there may be some nodes, or queues of nodes, that are more suitable for your tasks than others.
+
+## Choosing a queue
+
+At present, the new cluster only has one queue for general use, called `AllNodes` containing, as it says, all the available compute nodes. At other times though, during workshops for example we have run a `Training` queue, with strict limits, to ensure we've had capacity to demonstrate cluster use in a live setting.
+
+It also may be necessary in the future to partition the set of nodes, either by their capabilities if that becomes significant to some users, or by which research group might have purchased them, or to allow a particular group more protected access for a period.
+
+Use `hipercow_cluster_info` to see what queues are available, and then pass one of those into the `queue` parameter for `hipercow_resources`, to ensure tasks run on a particular queue.
+
+## Choosing nodes
+
+Even more rarely, you may have a particular node you want to run on. In the past, for instance, we have had specific nodes with specific hardware (such as very large RAM or large disks). Also occasionally, Wes or Rich might want to rerun tasks on a specific node if we want to replicate a failure that occurred on it, and diagnose whether there is a problem on that node.
+
+The list of nodes is provided by `hipercow_cluster_config()`, and you can specify a vector of strings for the node names that you are happy for your tasks to run on, as the `requested_nodes` parameter for `hipercow_resources`.
```

Comment:
+ tiny non-run example?

Reply:
Done - also hipercow_cluster_config was a lie - it's _info